cluster/images/provider-gcp/Dockerfile

```diff
@@ -9,4 +9,5 @@ ADD provider /usr/local/bin/crossplane-gcp-provider
 COPY stack-package /
 
 EXPOSE 8080
+USER 1001
```

Comment:
@hasheddan did you get an understanding of the mapping between container user and host user? i.e. what does it mean if user 1001 doesn't exist on the host, does that even matter?

Reply:
@jbw976 I am testing with templating controller now. `USER 1000` will usually work, but we can't know for certain every scenario that someone could run this container so it theoretically could still get mapped to root. In that case the Crossplane stack manager would reject it and the user would need to work out their strange config.

Reply:
ref: https://github.com/crossplane/templating-controller/pull/14